rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/LRACoordinatorRecovery1TestCase.java

```diff
@@ -127,6 +128,7 @@ public void after() {
      * @throws URISyntaxException if the LRA or recovery URIs are invalid (should never happen)
      */
     @Test
+    @Ignore
```

Comment:
It was an oversight while I was testing my changes to speed up my testing. However, the intention is to delete the test since I ported it to use undertow instead of wildfly (https://github.com/mmusgrov/narayana/blob/JBTM-3397-nesting/rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/LRATest.java#L676).

And I intended to follow up with a similar port of LRACoordinatorRecovery2TestCase which is `@Ignore`d because it had transient failures on our CI (which we know struggles with some tests).

Reply:
https://issues.redhat.com/browse/JBTM-3419 is moving this test to the artefact lra-test-basic